E2E/README.md

```diff
@@ -1,4 +1,206 @@
 # Protractor
-> Best practices using Protractor
+> Some aspects for Protractor usage in projects
 
-## In construction
+
+## Table of contents
```

Comment:
There are some categories here that are child of "Best Practices". Could you nest them in the TOC?